mlflow/tracking/fluent.py

```diff
@@ -371,6 +389,8 @@ def get_artifact_uri(artifact_path=None):
     run will be returned; calls to ``log_artifact`` and ``log_artifacts`` write
     artifact(s) to subdirectories of the artifact root URI.
 
+    **Note:** If no run is active, it will instantiate a new run.
```

Comment:
+1 thanks for this fix. It looks like we never mentioned that this method was side-effecting at all, which is pretty scary :P, hopefully this helps reduce some of the surprise encountered when calling it.